presto-product-tests/src/main/java/io/prestosql/tests/hive/TestCsv.java

```diff
@@ -62,12 +63,14 @@ private void testInsertIntoCsvTable(String tableName, String additionalTableProp
     }
 
     @Test(groups = STORAGE_FORMATS)
+    @Flaky(issue = "https://github.com/prestosql/presto/issues/4936", match = "Error committing write to Hive(?s.*)could only be replicated to 0 nodes instead of minReplication")
```

Comment:
extract constants?

Reply:
pre-existing. Also, I am not convinced. I'd rather see the problem fixed rather than make it easier to _update_. And in the worst case, updating multiple occurrences is not a big deal.